tests/boards/mtk_adsp/src/main.c

```diff
@@ -81,6 +79,8 @@ static void mbox_fn(const struct device *mbox, void *arg)
  */
 ZTEST(mtk_adsp, mbox)
 {
+	/* Different SOCs transmit the replies on different devices!  Just listen to both */
```

Comment:
For those curious: yes, this happened without an ABI bump in the SOF kernel driver.  Not hard to accommodate both, even in real audio firmware, but.... ew.